apps/web/src/routes/NonAuthLayoutRouter/NonAuthLayoutRouter.js

```diff
@@ -63,22 +63,22 @@ export default function NonAuthLayoutRouter (props) {
   }, [dispatch, setReturnToPath, returnToPath])
 
   return (
-    <Div100vh className={classes.nonAuthContainer}>
+    <Div100vh className='w-full h-full'>
```

Comment:
im seeing this no longer has overflow-y: auto, have you tested this on public group landing pages that may have content more than one page? i will do that